### DISCUSSION TITLE
Dig into Enterprise-Scale architecture and design methodoloy

### DISCUSSION THREAD

**Discussion by krnese** [Azure/Enterprise-Scale#393] [Show and tell]
We've recorded a demo that walks through some of the design principles of Enterprise-Scale, such as subscription democratization and policy driven governance, and how that help organizations to achieve autonomy for their business.

Recording available [here](https://www.youtube.com/watch?v=KjGlmud0U0Y&feature=youtu.be)

**Comment by angerfirecn** [comment]
Cool！ very usefull!
